redshift_sqlalchemy/dialect.py

```diff
@@ -145,19 +187,91 @@ class RedshiftDialect(PGDialect_psycopg2):
         }),
         (schema.Table, {
             "ignore_search_path": False,
-            'diststyle': None,
-            'distkey': None,
-            'sortkey': None
+            "diststyle": None,
+            "distkey": None,
+            "sortkey": None,
+            "sortkey_type": COMPOUND_SORTKEY,
         }),
     ]
 
     @reflection.cache
+    def get_table_options(self, connection, table_name, schema, **kw):
```

Comment:
I was using `*kw` to match the style of the function this is overriding. That seems to be pretty standard style in the SQLAlchemy code base. Would you prefer we just use `**kwargs` everywhere?